index.html

```diff
@@ -26,9 +26,10 @@
     <div class="reveal">
       <div class="slides">
         <section id="qrcode" data-menu-title="QR code">
-          <div class="qrcode" id="qrcode-talk"/>
```

Comment:
Er, that `div` *is* closed correctly (it's a singleton).

If that does break something for you, I wonder if
```html
<div class="qrcode" id="qrcode-talk"></div>
```
... would un-break that too?